Added option to use "Full" Chirp

genestealer opened the pull request:
Added and tested option to library to use different delays in the two I2CSoilMoistureSensor functions, depending if you are using the I2C or "full" chirp sensor.

genestealer committed:
Added variable delay for chirp sensor

genestealer committed:
Updated

genestealer committed:
Update README.md

genestealer committed:
Update README.md

genestealer committed:
Update README.md

genestealer committed:
Update README.md

Apollon77 commented:
Great. Can you also include anything forks 1.2.0? Then I can faster release that.

genestealer commented:
Sorry not sure I understand what you are asking.

Apollon77 commented:
Ol,  checked back, in fact only library.properties is missing to make it a 1.2.0 :-) Will check if I find time these days

genestealer committed:
Added note, Chirp! is without a thermistor

Sensor version: https://github.com/Miceuz/i2c-moisture-sensor/blob/master/i2c-moist-sensor-schematics.pdf
Chirp! version: https://github.com/Miceuz/PlantWateringAlarm/blob/master/schematics.png

Apollon77 commented:
Hey,, sorry for that mega delay ... that PR slipped through somehow.

To be honest I do not like those "static hard coded wait/delay times" ... why they are that way? why hard coded?
